src/olympia/devhub/templates/devhub/includes/addons_edit_nav.html

```diff
@@ -30,7 +30,7 @@
           <a href="{{ url }}">{{ title }}</a></li>
       {% endfor %}
 
-      {% if addon.promoted_subscription and addon.promoted_subscription.stripe_checkout_completed %}
+      {% if addon.promoted_subscription and addon.promoted_subscription.is_active == True %}
```

Comment:
```suggestion
      {% if addon.promoted_subscription and addon.promoted_subscription.is_active %}
```
(it's only `False` and `None` that need disambiguating)

Reply:
or, if you really want to be explicit, `is True`